contracts/controller/src/views.rs

```diff
@@ -50,12 +52,14 @@ impl Contract {
                 let total_borrows = self.get_total_borrows(user_id.clone());
                 let total_supplies = self.get_total_supplies(user_id.clone());
                 let health_factor = self.get_health_factor(user_id.clone());
+                let user_profile = self.user_profiles.get(user_id).unwrap();
```

Comment:
what if user_profile won't be found? If you implemented a Default state, so you can use unwrap_or_default

unwrap could cause a panic effect, not sure that it good manner

Reply:
we have the check above that account has to be present in the user_profiles `filter(|user_id| self.user_profiles.get(user_id).is_some())` so i suppose there are no case that we will be trying to get account that is not exist